### ISSUE TITLE
Clarify meaning of relative paths SPM plugin 'protoFiles'

### ISSUE THREAD

**Issue by Jerry-Carter** [grpc/grpc-swift#1487]
In the [SPM plugin documentation](https://github.com/grpc/grpc-swift/blob/main/Sources/protoc-gen-grpc-swift/Docs.docc/spm-plugin.md), the 'protoFiles' are listed as relative to the target source directory.  This is only partially true (as least as I think of relative paths).

Consider

"protoFiles" : [
    "Local.proto",
    "Model/Child.proto",
    "../Parent.proto",
    "../Common/Sibling.proto"
]

The first two work perfectly.  The later two do not appear to be supported.

Would it be possible to add a note to the documentation to prevent any confusion by others?

**Comment by glbrntt** [comment]
You are correct: the last two are not supported. The files must be relative to the root of the target source directory _and_ contained within the target source directory.
